internal/stacks/fftm_config.go

```diff
@@ -0,0 +1,123 @@
+// Copyright © 2021 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stacks
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/hyperledger/firefly-cli/pkg/types"
+	"github.com/miracl/conflate"
+	"gopkg.in/yaml.v2"
+)
+
+type FFTMLogConfig struct {
+	Level string `yaml:"level,omitempty"`
+}
+
+type FFTMAPIConfig struct {
+	Address string `json:"address"`
+	Port    int64  `json:"port"`
+}
+
+type FFTMConnectorConfig struct {
+	URL     string `json:"url"`
+	Variant string `json:"variant,omitempty"`
+}
+
+type FFTMFFCoreConfig struct {
+	URL string `json:"url"`
+}
+
+type FFTMManagerConfig struct {
+	Name string `json:"name"`
+}
+
+type FFTMPolicyEngineConfig struct {
+	Name   string                       `json:"name"`
+	Simple FFTMSimplePolicyEngineConfig `json:"simple"`
+}
+
+type FFTMSimplePolicyEngineConfig struct {
+	FixedGasPrice string
```

Comment:
Should this have a `json` tag here?

Reply:
Realized when doing the signer, these should all be `yaml`